prody/proteins/interactions.py

```diff
@@ -867,14 +865,31 @@ def calcProteinInteractions(atoms, **kwargs):
     return AllInteractions
 
 
-def calcHydrogenBondsDCD(atoms, trajectory, distA=3.0, angle=40, cutoff_dist=20, **kwargs):   
+def calcHydrogenBondsDCD(atoms, trajectory=None, distA=3.0, angle=40, cutoff_dist=20, **kwargs):   
     """Compute hydrogen bonds for DCD trajectory using default parameters.
         
     :arg atoms: an Atomic object from which residues are selected
     :type atoms: :class:`.Atomic`
         
-    :arg trajectory: DCD filename
-    :type trajectory: str"""
+    :arg trajectory: trajectory file
+    :type trajectory: class:`.Trajectory`
+
+    :arg distA: non-zero value, maximal distance between donor and acceptor.
+    :type distA: int, float, default is 3.0
```

Comment:
have you checked that putting the default like this works yet?

Reply:
It might be that we can't check that without building the website, but maybe there's another way

Reply:
It was working fine, but I changed it. Now all is in kwargs.